src/main/java/servlets/ReceiptAnalysis.java

```diff
@@ -123,6 +125,7 @@ private static AnalysisResults analyzeImage(ByteString imageBytes) throws IOExce
 
     // Generate categories and parse date and price if text was extracted.
     if (analysisBuilder.getRawText().isPresent()) {
+      System.out.println(analysisBuilder.getRawText().get());
```

Comment:
Could we use a logger as was done here: https://github.com/googleinterns/receipt-roundup/blob/b51c880fa819564c35e534761f8a6f811807f47a/src/main/java/servlets/EditReceiptServlet.java#L48

Reply:
Yeah, this was just a debugging statement that accidentally got merged in.